tests/drivers/uart/uart_async_api/testcase.yaml

```diff
@@ -20,6 +20,7 @@ common:
     - wio_terminal
     - xiao_esp32c3
     - atsamd20_xpro
+    - ev11l78a
```

Comment:
The exclusion won't be needed if #61193 makes it to the tree before this pull request.